content/_data/events/2018-12-18-jenkins-online-meetup.adoc

```diff
@@ -0,0 +1,7 @@
+---
+title: "Jenkins Online Meetup"
+date: “2018-12-18T17:00:00”
```

Comment:
Oh BTW I think there's a mistake here actually? If this is UTC, then it should be 16:00 right?

Reply:
fixed